$NetBSD: patch-lib_tcl_xcircuit.sh.in,v 1.1 2018/02/08 09:43:32 hauke Exp $

TERM may be empty, resulting in a spurious
[: cygwin: unexpected operator

--- lib/tcl/xcircuit.sh.in.orig	2014-05-17 14:15:47.000000000 +0000
+++ lib/tcl/xcircuit.sh.in
@@ -11,7 +11,7 @@ XCIRCUIT_WISH=WISH_EXE
 export XCIRCUIT_WISH
 
 # Hacks for Cygwin
-if [ ${TERM:=""} = "cygwin" ]; then
+if [ x${TERM:=""} = x"cygwin" ]; then
    export PATH=$PATH:TCLLIBDIR
    export DISPLAY=${DISPLAY:=":0"}
 fi
